### PR TITLE
Prepare v1.3.0 release

### PR DESCRIPTION
**Pull request by MikeGoldsmith**
## Which problem is this PR solving?
Prepares the v1.3.0 release.

## Short description of the changes
- Adds changelog entry (plus some tidy up for consistency)
- Updates version to 1.3.0

### CODE REVIEW

File: CHANGLOG.md [hunk 1]
```diff
@@ -1,8 +1,31 @@
...
+- Bump Microsoft.Extensions.Http from 2.1 to 6.0.0 (#51)
+- Bump Microsoft.AspNetCore.Http from 2.1 to 2.2.2 (#53)
+- Bump Moq from 4.10.1 to 4.16.1 (#54)
+- Bump Microsoft.Extensions.Logging from 2.1 to 6.0.0 (#46)
+- Bump Microsoft.Extensions.Hosting from 2.1 to 6.0.0 (#47)
+
 ## v1.2.0
 
-### Inmprovements
+### Inprovements
```
Comment (JamieDanielson) [inline CHANGLOG.md:28]:
```suggestion
### Improvements
```

😃

File: CHANGLOG.md [hunk 2]
```diff
@@ -1,8 +1,31 @@
 # libhonet-dotnet Changelog
 
+## v.1.3.0
```
Comment (JamieDanielson) [inline CHANGLOG.md:3]:
```suggestion
## v.1.3.0 2022-04-12
```

even though there weren't dates added in the past, should we start adding now?

**Review by JamieDanielson — APPROVED** [review]
👍 
looks like we haven't added dates on releases in the past, suggestion to add one now 🤷

File: CHANGLOG.md (hunk 1 above)
Comment (MikeGoldsmith) [reply CHANGLOG.md:28]: Dang it!

**Pull request merged**